modules/tpm2-tss

```diff
@@ -12,10 +12,11 @@ tpm2-tss_configure := aclocal && automake --add-missing && autoreconf -fi \
 	$(CROSS_TOOLS) \
 	--host $(MUSL_ARCH)-elf-linux \
 	--prefix "/" \
+	--with-sysroot=$(INSTALL) \
```

Comment:
Are you saying we should set the sysroot for all modules in the global Makefile @tlaurion ?

Reply:
@JonathonHall-Purism Ideally anything common and forced in modules should be in Global Makefile, I think?
-g0 -Os and everything else fixating properly pkg-config and sysroot should be passed along to builds.

Ideally, modules configs should
CFLAGS+="needed additional flags for build"
LDFLAGS+="needed additional link param for build"

Where everything common should be stripped of the modules to ease development. 
@JonathonHall-Purism any insights to do this properly, Makefile foo master? :)

Reply:
`-g0 -Os` were added and then removed in a couple of different commits, this is just the final outcome without the intermediate commits. I'd like to do makefile re-orgs in a different PR instead of growing this one too large.